configs/fischertechnik_TXT_rootfs_defconfig

```diff
@@ -94,6 +95,7 @@ BR2_PACKAGE_USB_MODESWITCH_DATA=y
 BR2_PACKAGE_USBMOUNT=y
 BR2_PACKAGE_USBUTILS=y
 BR2_PACKAGE_PYTHON3=y
+BR2_PACKAGE_PYTHON3_READLINE=y
```

Comment:
why do we need this?

Reply:
it should allow for tab completion in interactive python